docs/serving/_index.md

```diff
@@ -59,27 +59,27 @@ matching route and configuration created. Each time the `Service` is updated, a
 new revision is created.
 
 For more information on the resources and their interactions, see the
-[Resource Types Overview](https://github.com/knative/serving/blob/main/docs/spec/overview.md)
+[Resource Types Overview](https://github.com/knative/specs/blob/main/specs/serving/overview.md)
 in the Knative Serving repository.
 
 ## More samples and demos
 
-- [Knative Serving code samples](./samples/README.md)
+- [Knative Serving code samples](./samples/)
 
 ## Debugging Knative Serving issues
 
-- [Debugging Application Issues](./debugging-application-issues.md)
+- [Debugging Application Issues](./debugging-application-issues)
 
 ## Configuration and Networking
 
-- [Configuring cluster local routes](./cluster-local-route.md)
-- [Using a custom domain](./using-a-custom-domain.md)
-- [Assigning a static IP address for Knative on Google Kubernetes Engine](./gke-assigning-static-ip-address.md)
-- [Using subroutes](./using-subroutes.md)
+- [Configuring cluster local routes](./cluster-local-route)
+- [Using a custom domain](./using-a-custom-domain)
+- [Assigning a static IP address for Knative on Google Kubernetes Engine](./gke-assigning-static-ip-address)
```

Comment:
Huh, this is a weird one (since it looks to be provider-specific).

In any case, I'd move this over to the admin guide one of these years. 😁 

NOT FOR THIS PR!